チャット画面の編集

Yoshino-Morita opened the pull request:
# WHAT
index.html.hamlの作成
index.scssの作成

#WHY
チャット画面の作成のため

Commit:
Create  index.html.haml

naaaaya commented:
scssファイルの名前はモデルに基づいて作成しましょう。
index.scssだと、他のモデルにもindexが存在するため混乱してしまいます。

naaaaya commented:
ビューのスクリーンショットを添付いただけますでしょうか。
今後ビューに関するプルリクの際は画像も添付してください。

Commit:
Write index.scss & index.hrml.haml

# what
最初のレイアウトを考えるために、htmlとcssを組んだ

# why
チャットスペースのフロント画面実装のため
※前回モデルを作るようにと指示されたのですが、現段階では見た目を仮置きで組むとテキストに書かれております。どのようにしたらいいでしょうか。

naaaaya commented on app/views/messages/index.html.haml:
```diff
@@ -1,13 +1,39 @@
-
-
+    .chat-side__contents
+      %ul.chat-side__groups
```
sidebarは部分テンプレートで記述しましょう。

naaaaya commented on app/views/messages/index.html.haml:
```diff
@@ -1,13 +1,39 @@
+    .chat-main__title   Sampleグループ
+    .chat-main__member  Members:sample,sample
+    .chat-main__contents
+      %ul.chat-main__post
```
今はメッセージ仮置きで大丈夫です。

naaaaya reviewed (commented):
グループのメンバーの欄が下に詰まっているので
下にmarginをつけたほうがより良いと思います。

naaaaya commented:
実装が完了した後はWIPを外しましょう。

Commit:
チャット画面の編集

# whart
部分テンプレートの作成
# why
チャット画面作成のため
# question
テキスト入力部分が最下部に置けない。
#next
font-awesomeの導入

sho-kanamaru commented on app/assets/stylesheets/application.scss:
```diff
@@ -1 +1,3 @@
 @import "./reset";
+@import "./index";
+@import font -awesome
```
@import "font-awesome";で読み込みましょう！

sho-kanamaru commented on app/assets/stylesheets/index.scss:
```diff
@@ -0,0 +1,128 @@
+$side-message-color: #FFFFFF;
+$main-message-color: #434A54;
+$side-content-width: 300px;
+$content-height:calc(100vh - 106px);
```
変数化したものは別のファイルで管理しましょう！
変数化したファイルをapplication.scssでインポートするようにしましょう！

sho-kanamaru commented:
せっかくBEMで命名しているので、scssファイルは、& （アンパサンド）を使用してネストさせることで効率的に記述しましょう！

Commit:
チャット画面の実装

# what
関数を部分テンプレートにまとめる
Font-awesomeの導入
# why
チャット画面作成のため
# question
テキスト入力位置の調整

Yoshino-Morita commented:
<img width="1440" alt="2017-07-04 18 36 51" src="https://user-images.githubusercontent.com/29768608/27824769-79e39396-60e8-11e7-87a5-3cd96c20e78c.png">

sho-kanamaru commented on app/assets/stylesheets/application.scss:
```diff
@@ -1 +1,6 @@
+@import "font-awesome";
+@import "./index";
+
+
```
余計な改行は消しましょう！

sho-kanamaru commented:
前回コメントした& （アンパサンド）を使用してネストさせる部分が修正されていませんので、確認してください！以下のカリキュラムを参考にしてください！
http://master.tech-camp.in/expert/curriculums/1091

Commit:
チャット画面の作成

# WHAT
部分テンプレの改行削除
SCSSにて＆を使用した書き直し
テキスト入力欄の修正

# WHY
フロント画面実装のため

Yoshino-Morita commented:
<img width="1440" alt="2017-07-04 19 46 10" src="https://user-images.githubusercontent.com/29768608/27827354-cec55102-60f1-11e7-8be0-414c10423e42.png">
